CHANGELOG.md

```diff
@@ -9,6 +9,29 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.4.0 - 2019-02-20
+
+### Added
+
+- Add support for new encapsulation modes: noEncap (inter-Node Pod traffic is never encapsulated) and hybrid (inter-Node Pod traffic is only encapsulated if Nodes are not on the same subnet).
+- Add support for "named ports" to Network Policy implementation.
+- Add user documentation for IPsec support.
+- Add support for new "table" output mode to antctl.
```

Comment:
I hoped to have agent-info command in. But do not know why @lzhecheng did not merge his PR.

Reply:
Added